Movies/Models/MovieModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MovieModel.swift
+//  Movies
+//
+//  Created by Tyler Green on 5/12/23.
+//
+
+import Foundation
+
+struct SearchResults: Codable {
+    let results: [MovieModel]
+}
+
+struct MovieModel: Codable {
+    let id: Int
+    var title: String
+    
+    init(id: Int, title: String) {
```

Comment:
Swift will provide an automatic init for structs so you dont need to implement this unless you are accessing this struct in a separate framework